src/components/SocialShare/renderers/point-breakdown.tsx

```diff
@@ -65,9 +65,8 @@ export const PointBreakdown = (props: PointBreakdownProps) => {
                 style={[
                   {
                     backgroundColor:
-                      point.trace.color ?? i === 0
-                        ? theme.colors.primary
-                        : theme.colors.secondary,
+                      point.trace.color ??
+                      (i === 0 ? theme.colors.primary : theme.colors.secondary),
```

Comment:
This made a functional difference?  I'm a little surprised and might learn something if so.

Reply:
Yeah this was a bit tricky, before it was interpreting it as `(point.trace.color ?? i) === 0 ? ...` which made it evaluate to `false` if `point.trace.color` was defined.

Reply:
Interesting!